packages/ui-components/src/components/Tooltip.tsx

```diff
@@ -13,14 +13,19 @@ const TOOLTIP_COLOR = '#002d47';
 // https://github.com/mui-org/material-ui/issues/11467
 
 // For placement options @see https://material-ui.com/api/tooltip
-export const Tooltip = styled((props: TooltipProps) => (
-  <MuiTooltip
-    {...props}
-    classes={{ popper: props.className }}
-    placement={props.placement || 'top'}
-    arrow
-  />
-))`
+export const Tooltip = styled(
+  ({ className, placement = 'top', enterDelay = 1000, ...props }: TooltipProps) => (
```

Comment:
this will add a delay everywhere including tupaia-web which I'm not sure we want. The tooltip is only used in two places in datatrak-web so can we just set the delay as a constant in there and pass it into here?